source/arbiter.rst

```diff
@@ -36,3 +50,40 @@ with 4 data instances and 1 Arbiter:
        enabled: true
        size: 1
        ....
+
+.. note:: You can find description of other possible options in the :ref:`replsets.arbiter section<replsets-arbiter-enabled>` of the :ref:`Custom Resource options reference<operator.custom-resource-options>`.
+
+Adding non-voting nodes
+-----------------------
+
+`Non-voting member <https://docs.mongodb.com/manual/tutorial/configure-a-non-voting-replica-set-member/>`_
+is a Replica Set node which does not participate in the primary
+election process. This feature is required if having more than 7 nodes, or if
+there is a `node in the edge location <https://en.wikipedia.org/wiki/Edge_computing>`_,
+which obviously should not participate in the voting process.
+
+Percona Distribution for MongoDB Operator has the ability to configure non-voting
+nodes in the Replica Set section of the
+`deploy/cr.yaml <https://github.com/percona/percona-server-mongodb-operator/blob/main/deploy/cr.yaml>`_
+file:
+
+-  set ``nonvoting.enabled`` option to ``true`` to allow non-voting instances,
+-  use ``nonvoting.size`` option to set the desired amount of non-voting instances.
+
+For example, the following keys in ``deploy/cr.yaml`` will create a cluster
+with 3 data instances and 1 non-voting instance:
+
+.. code:: yaml
+
+   ....
+   replsets:
+     ....
+     size: 3
+     ....
+     nonvoting:
+       enabled: true
+       size: 1
+       ....
+
+.. note:: You can find description of other possible options in the :ref:`replsets.nonvoting section<replsets-nonvoting-enabled>` of the :ref:`Custom Resource options reference<operator.custom-resource-options>`.
+
```

Comment:
Lets also add a link to replication with smth like "it is possible to add a non-voting node in the edge location through externalNodes option. Please refer to replication doc". You mention edge below, maybe there a good place as well